crates/aptos/src/move_tool/aptos-natives.bpl

```diff
@@ -0,0 +1,2 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
Is this supposed to have something in it?

Reply:
It's an empty template. Things will be added as we go.